.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+
+python:
+    - 3.6
+    - 3.7
+    - 2.7
+
+install:
+  - pip install -e .
+
+script:
+  - cif2cell --help
+  - .ci/test_script.sh
+  - pytest tests
```

Comment:
Does this pass on travis?
Probably need to add a `tests` extra to the `setup.py` in order to get `pytest` and `pip install -e .[tests]`.